### ISSUE TITLE
Forbidden sub-reddit

### ISSUE THREAD

**Issue by dominicglenn** [michael-lazar/rtv#119]
When trying to access a specific sub-reddit (most likely non-existent as well), a forbidden error is returned.

The sub-reddit in questions was 'aaa':

```
Connecting...
Traceback (most recent call last):
  File "/usr/local/bin/rtv", line 7, in <module>
    from rtv.__main__ import main
  File "/Library/Python/2.7/site-packages/rtv/__main__.py", line 138, in <module>
    sys.exit(main())
  File "/Library/Python/2.7/site-packages/rtv/__main__.py", line 118, in main
    page.loop()
  File "/Library/Python/2.7/site-packages/rtv/subreddit.py", line 49, in loop
    self.controller.trigger(cmd)
  File "/Library/Python/2.7/site-packages/rtv/page.py", line 225, in trigger
    return func(self.instance, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/rtv/subreddit.py", line 92, in prompt_subreddit
    self.refresh_content(name=name)
  File "/Library/Python/2.7/site-packages/rtv/subreddit.py", line 58, in refresh_content
    self.reddit, name, self.loader)
  File "/Library/Python/2.7/site-packages/rtv/content.py", line 313, in from_name
    return cls(display_name, submissions, loader)
  File "/Library/Python/2.7/site-packages/rtv/content.py", line 258, in __init__
    self.get(0)
  File "/Library/Python/2.7/site-packages/rtv/content.py", line 327, in get
    submission = next(self._submissions)
  File "/Library/Python/2.7/site-packages/praw/__init__.py", line 524, in get_content
    page_data = self.request_json(url, params=params)
  File "/Library/Python/2.7/site-packages/praw/decorators.py", line 173, in wrapped
    return_value = function(reddit_session, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/praw/__init__.py", line 579, in request_json
    retry_on_error=retry_on_error)
  File "/Library/Python/2.7/site-packages/praw/__init__.py", line 424, in _request
    _raise_response_exceptions(response)
  File "/Library/Python/2.7/site-packages/praw/internal.py", line 196, in _raise_response_exceptions
    raise Forbidden(_raw=response)
praw.errors.Forbidden
```

**Comment by michael-lazar** [comment]
Works for me, can you check what version of RTV and PRAW you're using?


![image](https://cloud.githubusercontent.com/assets/5026795/8704453/e046f392-2ad7-11e5-84fd-9300f615fa2e.png)

**Comment by dominicglenn** [comment]
praw==3.1.0
rtv==1.4.1

**Comment by michael-lazar** [comment]
Yup. That latest version of praw added some new exception types, and we weren't catching them properly. Thanks for bringing this up!

Fixed in b913990
